sdk/agrifood/azure-verticals-agrifood-farming/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.0.0-beta.3 (Unreleased)
 
+- Adding clients for Sensor Integration which includes crud operations on DeviceDataModels, Devices, SensorDataModels, Sensors, SensorMappings, SensorPartnerIntegration and get Sensor events.
+- Renaming farmers to parties as per the new API version.
+- Renaming Crop Variety to Crop Product.
+- Adding new APIs for STAC search.
```

Comment:
double check https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/8/Release-Checklist

e.g. you changelog is almost invalid.

breaking changes need to be listed.

Reply:
Updated.